src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../styles/themes";
+
+.container,
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.container {
+  z-index: 2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container[aria-hidden="true"] {
+  display: none;
+
+}
+
+.overlay {
+  background-color: rgba(41, 47, 76, 0.7);
+}
+
+.dialog {
+  z-index: 2;
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  padding: 0 30px;
+  padding-block-end: 24px;
+  border-radius: 16px;
```

Comment:
Probably we should use design-tokens here:
https://github.com/mondaycom/monday-ui-style/blob/master/src/config/tokens/core/_core-radius.scss

Reply:
`var(--spacing-large)` would be even better